otter/log/formatters.py

```diff
@@ -127,16 +129,17 @@ def PEP3101FormattingWrapper(observer):
     :rtype: :class:`ILogObserver`
     """
     def PEP3101FormattingObserver(eventDict):
-        if eventDict.get('why'):
+        if (eventDict.get('why') and
+                eventDict.get('system') not in NON_PEP3101_SYSTEMS):
             eventDict['why'] = eventDict['why'].format(**eventDict)
 
         if 'message' in eventDict:
             message = ' '.join(eventDict['message'])
 
-            if message:
+            if message and eventDict.get('system') not in NON_PEP3101_SYSTEMS:
```

Comment:
Couldn't this new conditional clause be moved up to the `if 'message' in eventDict` conditional?